### PR TITLE
New Site Overhaul

### PR DESCRIPTION
**Pull request by PardusEidolon**
# Changelog 🚀
- Complete site overhaul with new visuals and fonts
- Runs using the [doks](https://getdoks.org/) template using `npm` to manage hugo
- light and dark themes
- Currently content only has support for Indonesian, Vietnamese, Japanese languages

### Remarks
_kudos for @jennyberry for going ahead an creating a new site template for the eastern townhall website! 👏_

### CODE REVIEW

File: config/_default/params.toml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+alert = false
+alertDismissable = true
+# alertText = "Introducing the Doks child theme, several DX + UX updates, and more! <a class=\"alert-link stretched-link\" href=\"https://getdoks.org/blog/doks-v0.2/\" target=\"_blank\" rel=\"noopener\">Check out Doks v0.2</a>"
+alertText = "Introducing the Doks child theme, several DX + UX updates, and more! <a class=\"alert-link stretched-link\" href=\"https://getdoks.org/blog/doks-v0.2/\">Check out Doks v0.2</a>"
+
+# Edit Page
+# repoHost [Github | Gitea | GitLab | Bitbucket | BitbucketServer ] is used for building the edit link based on git hoster
+repoHost = "GitHub"
+#repoHost = "Gitea"
+docsRepo = "https://github.com/C3ETH/c3eth.git"
```
Comment (jennyberry) [inline config/_default/params.toml:89]: do you need to change this link?

Comment (PardusEidolon) [reply config/_default/params.toml:89]: yes, most likely

Comment (PardusEidolon) [reply config/_default/params.toml:89]: are there any other Urls i should worry about? or is it just this one?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,5 +1,4 @@
-blank_issues_enabled: false
 contact_links:
```
Comment (jennyberry) [inline .github/ISSUE_TEMPLATE/config.yml:1]: should change info in this file to our link

File: config/_default/params.toml (hunk 1 above)
Comment (jennyberry) [inline config/_default/params.toml:89]: link to repo should change

**Review by jennyberry — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (PardusEidolon) [reply .github/ISSUE_TEMPLATE/config.yml:1]: ok sweet, done

**Comment by PardusEidolon** [comment]
fingers crossed this works @jennyberry 🤞🤞🤞

**Pull request merged**